docs/flows/flows.puml

```diff
@@ -70,33 +72,52 @@ autonumber "Bond:[00]"
 
 alice -> forum: create forum proposal
 alice <- guarantor: positive/negative feedback
-alice -> bondcontract: if (positive) create bond
+alice -> bondcontract: if (positive) create bond (specify reward mechanism)
 
-guarantor -> bondcontract: contribute
+guarantor -> bondcontract: contribute BIT tokens
+bondcontract <-- guarantor: ERC20 Fungible bond tokens (1:1 swap)
 
 community -> community: delegate votes
 
-alice -> snapshot: submit proposal
+alice -> snapshot: if (positive) submit proposal
+
+tokenholders -> snapshot: show proposals
+tokenholders <-- snapshot: proposals
+tokenholders -> snapshot: vote on open proposals
+tokenholders <- snapshot: vote
 
 bitdaoadmin -> bitdaotreasury: if (pass) allow funds
 bitdaotreasury -> grantstreasury: transfer funds
```

Comment:
Funds transfer seem to fall into two family of approaches:
1) Give all the funds; when the `bitdaoadmin` allows funds, the passed amount is transferred to the receiving treasury.
2) Pull release; when the `bitdaoadmin` allows funds, a limit is set allowing the receiving treasury to pull funds up to that limit. 

The difference being the locus of control for triggering the transfers.

Reply:
Good points. This should be captured in some spec with tradeoffs and guidelines.